src/Polytopes/Polyhedron/properties.jl

```diff
@@ -3,17 +3,28 @@
 ### Iterators
 ###############################################################################
 ###############################################################################
+"""
+    Polyhedra
+
+Empty type for selecting `Polyhedron` as return type for polyhedra.
+"""
 struct Polyhedra
 end
+
+"""
+    Points
+
+Empty type for selecting `Vector` as return type for points.
```

Comment:
This is of course off-topic for this PR, but: I don't get why you need these two empty types. Why don't you just accept `Polyhedron` resp. `Vector` instead?